### ISSUE TITLE
Feature Request: Re-import data from csv

### ISSUE THREAD

**Issue by TZwielehner** [actaillacq/Station_Offset#3]
Exporting the station / offset data is great. I wonder if it is possible to edit the data in the csv file and update the position of the respective points in qgis? we often have "approximate" placement of elements that we assign a constant offset to.

**Comment by TZwielehner** [comment]
Also, it would be very handy to get the direction (perpendicular to the alignment) at each point.

**Comment by actaillacq** [comment]
> Exporting the station / offset data is great. I wonder if it is possible to edit the data in the csv file and update the position of the respective points in qgis? we often have "approximate" placement of elements that we assign a constant offset to.

I will probably need a little more help understanding the process to see if it can be implemented. The CSV format ends up being a bit of a PITA because we don't have the relational elements that are added when the information is put in a database.

**Comment by actaillacq** [comment]
> Also, it would be very handy to get the direction (perpendicular to the alignment) at each point.

That should be doable.

**Comment by TZwielehner** [comment]
The process we have is that we have an alignment (railway) and are placing points along the track manually. After exporting to csv / Excel I would like to clean up the offsets (e. g. 1,25 to the right) and reimport the points to use it as a simple linear reference system.

**Comment by actaillacq** [comment]
I would probably use an offset curve from your alignment and use this reference as you are dropping the points and then turn on snapping to avoid the additional processing steps. The QGIS tool is a bit clunky doing this though I must admit. I would probably jump over to CAD for this operation.

So basically if I am understanding correctly you are trying to do the reverse operation of generating station offset- provide a station and directional offset and then get back out northing and easting?

**Comment by TZwielehner** [comment]
Yes, you understand correctly. Sometimes it is easier to approximate the position first and align it later instead of entering the station / offset values right away.
